src/utils.ts

```diff
@@ -166,7 +166,15 @@ function fromMultibase(idString: string): Id | undefined {
   if (codec == null) {
     return;
   }
-  const buffer = codec.decode(idString);
+  let buffer: Uint8Array;
+  try {
+    buffer = codec.decode(idString);
+  } catch (e) {
+    if (e instanceof SyntaxError) {
+      return;
+    }
+    throw e;
+  }
```

Comment:
This is still broken. Here is an example of a failing test.
```ts
  test('fromMultibase is throwing an error', async () => {
    expect(IdInternal.fromMultibase('zF4VfF3uRhSqgxTOOLONGxTRdVKauV9')).toBeUndefined();
    expect(IdInternal.fromMultibase('zF4VfxTOOSHORTxTV9')).toBeUndefined();
  })
```

I think the problem is that it's throwing just a normal `Error()` with the message `Non-base58btc character` so it's not getting caught by the catch block and returning undefined.

Problem is that any base is valid for decoding and each one will have it's own error message. we may have to catch all errors here and return with undefined.

@CMCDragonkai , @emmacasolin

Reply:
Could have sworn it was `SyntaxError`.

Can you confirm if it is actually throwing `Error`?

Reply:
Ok I checked, yea it does do this.

Well then we just need to change it to:

```
} catch (e) {
  return;
}
```

Reply:
@emmacasolin can you clean this up.

Reply:
Fix is on the same branch but in a different PR here: https://github.com/MatrixAI/js-id/pull/15